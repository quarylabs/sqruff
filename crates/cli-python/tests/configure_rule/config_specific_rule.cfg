[sqruff]
dialect = ansi
rules = AL01

