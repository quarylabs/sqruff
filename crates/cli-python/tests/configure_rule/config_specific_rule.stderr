== [tests/configure_rule/_example.sql] FAIL
L:   1 | P:  28 | AL01 | Implicit/explicit aliasing of table. [aliasing.table]
The linter processed 1 file(s).
All Finished
